tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -282,8 +290,14 @@ EOF
 	# that does not exist.
 	#
 	echo "::group::Check the service healthy"
-	kbs_ip=$(kubectl get -o jsonpath='{.spec.clusterIP}' svc "$KBS_SVC_NAME" -n "$KBS_NS" 2>/dev/null)
-	kbs_port=$(kubectl get -o jsonpath='{.spec.ports[0].port}' svc "$KBS_SVC_NAME" -n "$KBS_NS" 2>/dev/null)
+	if [ "${KBS_INGRESS}" = "nodeport" ]; then
+		kbs_ip=$(kbs_k8s_svc_host)
+		kbs_port=$(kbs_k8s_svc_port)
+	else
+		kbs_ip=$(kubectl get -o jsonpath='{.spec.clusterIP}' svc "$KBS_SVC_NAME" -n "$KBS_NS" 2>/dev/null)
```

Comment:
Is the ingress already up at this point? If so we should be able to simply use the `kbs_k8s_svc_{host,port}` for all scenarios, shouldn't we?

Reply:
(I mean it should be possible to avoid the extra if/else branch and we should simply rely on the function (unless I miss something in AKS setup, I don't have on to try it)

Reply:
@ldoktor that part I am not sure if the use of those functions will work as I do not have a AKS setup to test it

Reply:
@wainersm could you please verify that?

Reply:
Using the upstream nodeport deployment shouldn't be needed to change the logic of this function. 

I didn't see the nodeport deployment upstream but when I created the issue I was considering it would be as simple as tweaking the `_handle_ingress_nodeport` function and everything else would work fine.

Reply:
@wainersm so I did not understand for the case of AKS should I leave it like it is or should I sue the functions that @ldoktor suggest it